index.html

```diff
@@ -21,9 +21,9 @@ <h1>OpenRCT2</h1>
     <h2>Open source re-implementation of RollerCoaster Tycoon 2</h2>
     <ul id="buttons">
       <li>
-        <a class="link" href="https://github.com/OpenRCT2/OpenRCT2/releases/download/v0.0.3.1/OpenRCT2-0.0.3.1-windows.zip">Download Stable</a>
+        <a id="link" href="https://github.com/OpenRCT2/OpenRCT2/releases/download/v0.0.3.1/OpenRCT2-0.0.3.1-windows.zip">Download Stable</a>
         <div class="information">
-          <span class="version">0.0.3.1</span> — <a class="platform" href="#" title="Change platform…">Windows</a> — <span class="size">12.3 MB</span>
+          <span id="version">0.0.3.1</span> — <a id="platform" href="#" title="Change platform…">Windows</a> — <span id="size">12.3 MB</span>
```

Comment:
I don't think there is really a good reason for doing this. Please also drop this commit.